vlib/crypto/README.md

```diff
@@ -1,14 +1,76 @@
 ## Description:
 
-`crypto` is a module that...
+`crypto` is a module that exposes cryptographic algorithms to V programs.
 
-*TODO*
+Each submodule implements things differently, so be sure to consider the documentation
+of the specific algorithm you need, but in general, the method is to create a `cipher`
+struct using one of the module functions, and then to call the `encrypt` or `decrypt`
+method on that struct to actually encrypt or decrypt your data.
+
+This module is a work-in-progress. For example, the AES implementation currently requires you
+to create a destination buffer of the correct size to receive the decrypted data, and the AesCipher
+`encrypt` and `decrypt` functions only operate on the first block of the `src`.
+
+This module is based on Go's crypto packages. That documentation is here: https://pkg.go.dev/crypto
+
+The Go source code license is reprinted here:
+
+```Copyright (c) 2009 The Go Authors. All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are
+met:
+
+   * Redistributions of source code must retain the above copyright
+notice, this list of conditions and the following disclaimer.
+   * Redistributions in binary form must reproduce the above
+copyright notice, this list of conditions and the following disclaimer
+in the documentation and/or other materials provided with the
+distribution.
+   * Neither the name of Google Inc. nor the names of its
```

Comment:
That is not needed - the code is a port, not a direct copy. However, you can link to it if you want.

Reply:
Thanks. This text is in the repo as `readme.txt` and I thought it was desired, but I agree that it is not needed. I will remove it.